README.md

```diff
@@ -127,7 +127,8 @@ git clone https://github.com/aeraki-mesh/aeraki.git
 
 ### Install Istio, Aeraki and demo Applications
 ```bash
-m
+make install
```

Comment:
remove this line,  "make demo" should be enough

Reply:
done